### PR TITLE
fix SolveResponse to match expected

### PR DESCRIPTION
**Pull request by decanus**
fixes the SolveResponse to match what is actually expected by autopilot

### CODE REVIEW

**Comment by decanus** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by decanus** [comment]
recheck

File: crates/driver/openapi.yml [hunk 1]
```diff
@@ -333,41 +333,51 @@ components:
...
+                type: string
+                example: "123"
+              submissionAddress:
+                description: |
+                  The address that will be used to submit the solution.
+                $ref: "#/components/schemas/Address"
+              orders:
+                description: executed orders
+                type: array
+                items:
```
Comment (MartinquaXD) [inline crates/driver/openapi.yml:362]: `orders` is an object mapping a `OrderUid` to executed amounts.

Comment (decanus) [reply crates/driver/openapi.yml]: fixed

File: crates/driver/openapi.yml [hunk 2]
```diff
@@ -334,41 +334,48 @@ components:
...
-            additionalProperties:
-              $ref: "#/components/schemas/BigUint"
+            properties:
+              solutionId:
+                description: |
+                  The unique identifier of the solution.
+    
+                  This id is used to identify the solution when executing it.
+                type: string
+                example: "123"
```
Comment (squadgazzz) [inline crates/driver/openapi.yml:350]: nit:
```suggestion
                example: "1"
```

File: crates/driver/openapi.yml [hunk 3]
```diff
@@ -334,41 +334,48 @@ components:
...
+              score:
+                description: |
+                  The objective value of the solution.
+                type: string
+                example: "123"
```
Comment (squadgazzz) [inline crates/driver/openapi.yml:355]: nit:
```suggestion
                example: "100"
```

**Review by squadgazzz — APPROVED** [review]

**Review by Mateo-mro — APPROVED** [review]

File: crates/driver/openapi.yml [hunk 4]
```diff
@@ -334,41 +334,48 @@ components:
...
+                description: |
+                  The objective value of the solution.
+                type: string
+                example: "100"
+              submissionAddress:
+                description: |
+                  The address that will be used to submit the solution.
+                $ref: "#/components/schemas/Address"
+              orders:
+                type: object
```
Comment (sunce86) [inline crates/driver/openapi.yml:361]: https://github.com/cowprotocol/services/blob/main/crates/autopilot/src/infra/solvers/dto/solve.rs#L146

Order uid is missing

**Pull request merged**